middleware.js

```diff
@@ -1,25 +1,5 @@
 "use strict";
 
-const auth = require("basic-auth");
-
-const AppConstants = require("./app-constants");
-
-
-function checkSESAuth (user, pass) {
-  return user === AppConstants.SES_USERNAME && pass === AppConstants.SES_PASSWORD;
```

Comment:
Are we using `SES_USERNAME` or `SES_PASSWORD` still?


```sh
$ git grep SES_USERNAME
  # .env-dist:SES_USERNAME=""
  # app-constants.js:  "SES_USERNAME",
  # middleware.js:  return user === AppConstants.SES_USERNAME && pass === AppConstants.SES_PASSWORD;

$ git grep SES_PASSWORD
  # .env-dist:SES_PASSWORD=""
  # app-constants.js:  "SES_PASSWORD",
  # middleware.js:  return user === AppConstants.SES_USERNAME && pass === AppConstants.SES_PASSWORD;
```

Reply:
Good catch! We can remove them now.